rca/events/models.py

```diff
@@ -763,8 +764,8 @@ def get_ics_record(self):
             # Get date
             date = self.start_date + datetime.timedelta(days=day)
             # Get times
-            start_datetime = datetime.datetime.combine(date, self.start_date.time())
-            end_datetime = datetime.datetime.combine(date, self.end_date.time())
+            start_datetime = datetime.datetime.combine(date, self.start_time)
+            end_datetime = datetime.datetime.combine(date, self.end_time)
```

Comment:
Same problem as `start_time`.

Reply:
Resolved in https://github.com/torchbox/rca-wagtail-2019/pull/865/commits/26b2ee410ef2d5ef2d147cea795d4f44bdbc733d